naucse/models.py

```diff
@@ -263,9 +271,18 @@ def get_lesson(self, name, base_collection=None):
 
 
 class Navigation(Model):
-    def __init__(self, title, url):
-        self.url = url
+    """Represents navigation link.
+
+    Arguments:
+        title   the title of the link location
+        url     <lesson_type>/<lesson>
```

Comment:
It's not really a [URL](https://en.wikipedia.org/wiki/Uniform_Resource_Locator).